sdk/tests/conformance/more/unit.js

```diff
@@ -71,6 +71,44 @@ var __testFailCount__ = 0;
 var __testLog__;
 var __backlog__ = [];
 
+var getUrlOptions = (function() {
```

Comment:
I would absolutely like to see the two testing frameworks consolidated. That felt like a bigger task, though, an more appropriate for a separate patch.